dashboard/src/api/resources/Resources.js

```diff
@@ -43,12 +43,15 @@ export const providerLogos = Object.freeze({
 
 
 // var API_URL = "http://127.0.0.1:5000/data"; //localmode
-var API_URL = "//"+ window.location.hostname
+// var API_URL = "//"+ window.location.hostname
+var API_URL = "http://localhost:8181";
 
 if (typeof process.env.REACT_APP_API_URL != "undefined") {
   API_URL = process.env.REACT_APP_API_URL.trim();
 }
-export var PROMETHEUS_URL = "//"+ window.location.hostname+"/prometheus";
+// export var PROMETHEUS_URL = "//"+ window.location.hostname+"/prometheus";
+
+export var PROMETHEUS_URL = "http://localhost:8181/prometheus";
```

Comment:
since this is the same host as the API url have the api url be one variable and have prometheus url be API_URL + "/prometheus"